apps/condo/domains/property/scheema/Property.test.js

```diff
@@ -1,9 +1,9 @@
 const faker = require('faker')
 const { UUID_RE, DATETIME_RE } = require('@core/keystone/test.utils')
 
-const { makeClientWithRegisteredOrganization } = require('../../utils/testSchema/Organization')
-const { buildingMapJson } = require('../../constants/property.example')
-const { Property } = require('../../gql/Property')
+const { makeClientWithRegisteredOrganization } = require('../../../utils/testSchema/Organization')
+const { buildingMapJson } = require('@condo/domains/common/constants/property.example')
```

Comment:
Looks like `@condo/domains/common/constants/property.example` should be moed to `@condo/domains/property/constants/property.example` (property domain)